Source/Core/DolphinQt/FIFO/FIFOPlayerWindow.cpp

```diff
@@ -324,9 +329,12 @@ void FIFOPlayerWindow::OnFIFOLoaded()
   m_analyzer->Update();
 }
 
-void FIFOPlayerWindow::OnEarlyMemoryUpdatesChanged(bool enabled)
+void FIFOPlayerWindow::OnConfigChanged()
 {
-  FifoPlayer::GetInstance().SetEarlyMemoryUpdates(enabled);
+  Config::SetBase(Config::MAIN_FIFOPLAYER_EARLY_MEMORY_UPDATES,
+                  m_early_memory_updates->isChecked());
+
+  FifoPlayer::GetInstance().RefreshConfig();
```

Comment:
Instead of requiring this function to be called, it would be nice if you could make FifoPlayer use config callbacks to automatically detect when the setting has been changed. See AdmiralCurtiss's recent PRs for examples of how to do this.

Reply:
Changed.  Note that the fifoplayer is currently a singleton, statically allocated in the `GetInstance()` function, so its destructor won't be called at runtime and thus the callback won't be unregistered for a while, but I don't think that's going to be a problem.